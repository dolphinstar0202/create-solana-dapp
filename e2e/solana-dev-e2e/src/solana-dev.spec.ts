import { execCapture, formatOutput } from '@solana-developers/e2e-utils'

describe('solana-dev', () => {
  it('should print the help command', async () => {
    const output = execCapture(`npx --yes solana-dev@e2e --help`)
    // Trim all whitespace from the output to make the snapshot more readable
    const trimmed = formatOutput(output)
    expect(trimmed).toMatchSnapshot()
  })
})
