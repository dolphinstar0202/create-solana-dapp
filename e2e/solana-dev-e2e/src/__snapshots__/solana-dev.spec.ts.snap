// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`solana-dev should print the help command 1`] = `
"Usage: solana-dev [options] [command]
Options:
-V, --version output the version number
-h, --help display help for command
Commands:
doctor
help [command] display help for command"
`;
